src/checkers/checker_type.rs

```diff
@@ -25,13 +25,17 @@ pub struct Checker<Type> {
     pub popularity: f32,
     /// lemmeknow config object
     pub lemmeknow_config: Identify,
+    /// https://doc.rust-lang.org/std/marker/struct.PhantomData.html
+    /// Let's us save code lines when we don't need to use a type parameter
     pub _phatom: std::marker::PhantomData<Type>,
```

Comment:
PhantomData is Zero-sized type, so saving actual space, I don't think it save lines of code. 

> Adding a `PhantomData<T>` field to your type tells the compiler that your type acts as though it stores a value of type T, even though it doesn’t really.

It is used to mark the Type when we don't want to use it! so that compiler is satisfied with it's checks.